### PR TITLE
KitchenSink - Use saucectl native way for cypress

### PR DESCRIPTION
**Pull request by FriggaHel**

### CODE REVIEW

**Review by tianfeng92 — APPROVED** [review]

**Comment by tianfeng92** [comment]
LGTM until the pipeline passes :)

**Review by alexplischke — COMMENTED** [review]
Hm, to reflect the most current changes I think the setup needs to change for this one, since CI mode support was removed.

1. We'll need to update the config.yaml to the newest one.
2. Add cypress.json
3. Run in a regular image and not within ours.
4. Perhaps even move saucectl out of the picture, depends on what this pipeline is supposed to do.

**Comment by FriggaHel** [comment]
@alexplischke / @tianfeng92 I'm not sure if this is the correct approach and I'm not sure there is any alternative (currently) to `chown` (uid from `runner` & `seluser`)

File: tests/kitchen-sink-tests/.sauce/config.yml [hunk 1]
```diff
@@ -8,17 +9,21 @@ metadata:
   build: Release $CI_COMMIT_SHORT_SHA
 
 files:
```
Comment (alexplischke) [inline tests/kitchen-sink-tests/.sauce/config.yml:11]: `Files` isn't used anymore, so you can remove it.

File: tests/kitchen-sink-tests/.sauce/config.yml [hunk 2]
```diff
@@ -1,4 +1,5 @@
 apiVersion: v1alpha
+kind: Cypress
 metadata:
```
Comment (alexplischke) [inline tests/kitchen-sink-tests/.sauce/config.yml:3]: Metadata is now underneath `sauce`. Here's a full config example, hope that helps :) --> https://github.com/saucelabs/saucectl/blob/master/.sauce/cypress-native.yml

**Review by alexplischke — APPROVED** [review]

**Pull request merged**